Deploy/datares_test.py

```diff
@@ -0,0 +1,118 @@
+import base64
+import os
+import datetime
+from urllib.parse import quote as urlquote
+
+from flask import Flask, send_from_directory
+import dash
+from PIL import Image
+import dash_core_components as dcc
+import dash_html_components as html
+from dash.dependencies import Input, Output, State
+
+UPLOAD_DIRECTORY = "images/"
+
+if not os.path.exists(UPLOAD_DIRECTORY):
+    os.makedirs(UPLOAD_DIRECTORY)
+
+# create a route for downloading files directly:
+server = Flask(__name__)
+app = dash.Dash(server=server)
+app.title = 'Black and White Converter'
+
+
+@server.route("/download/<path:path>")
+def download(path):
+    """Serve a file from the upload directory."""
+    return send_from_directory(UPLOAD_DIRECTORY, path, as_attachment=True)
+
+
+app.layout = html.Div([
```

Comment:
In the future is it possible to separate the html/css stuff and the python stuff? It might help for readability and general organization/layout of the project as it grows